portal/src/texts/blog/kulturForDesignsystem/kulturForDesignsystem.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Kultur for designsystem
+description: En kort historie om Fremtinds designsystemkultur
+author: Glenn A. Brownlee
+publishDate: "2021.08.03"
+---
+
+En liten gruppe utviklere og en designer startet i løpet av sommeren 2015 å gå fra team til team i SpareBank 1-universet for å fortelle om en idé vi jobbet med. 
+Ideen hadde vokst frem gjennom en samling av læring fra inspirerende folk og tekniske fremskritt iblandet en god dose frustrasjon over fragmenterte designimplementasjoner, 
+treg time-to-market, unødvendig lav UX-kvalitet og gjentagende problemstillinger.
+
+En av disse mange inspirerende folkene hadde jeg truffet høsten før (13. oktober 2014) på en CSS-konferanse i New Orleans. 
+[Jamie Strachan](https://www.linkedin.com/in/jamiestrachan/), en utvikler fra CBC, holdt et foredrag om objektorientert CSS og forklarte noe så enkelt som at de hadde begynt 
+å spørre designere hva de kalte de forskjellige visuelle delene av webløsningen de designet. Dette hjalp utviklerne å navngi komponentene de kodet. 
+En lang tankerekke startet: Hvordan ville verden vært om designere og utviklere kunne jobbe tettere sammen og til og med snakke samme språk? 
+Kunne en designer raskt påvirke brukeropplevelsen i hele den implementerte profilen på en trygg og versjonerbar måte? Kunne man lage gjenbrukbare 
+UX-standarder for alle som utformet webløsninger innen samme merkevare?
+
+## En smartere måte
+Etter mye lesing, mange flere inspirerende samtaler og diskusjoner, forsøkte vi å overbevise andre utviklere og designere i SpareBank 1 
+om at vi kunne jobbe på en smartere måte til stor fordel for både brukerne og oss. Etter hvert begynte andre i industrien å kalle disse 
+prinsippene for “designsystem” og vi oppdaget plutselig en enda større gruppe likesinnede som jobbet med lignende ideer. 
+Vi hadde etter hvert fått nok moment til å kunne lage et designsystem som flere og flere team i SpareBank 1 både kunne bruke og ønsket å bruke.
+
+I stedet for å starte på ideen og spre den til teamene, kunne vi gjennomført en problemanalyse, forsøkt å få støtte fra ledelsen, 
+formet et mandat og sparket i gang et prosjekt som skulle formalisere en ny design- og utviklingsmetode og tilhørende verktøykasse som alle skulle bruke. 
+Det ville kanskje fungert til en viss grad. Men hvis man ønsker at noe så stort skal virkelig være levende og intuitivt, 
+må det påvirkes kontinuerlig av alle menneskene som skal etterleve det. Den kollektive kunnskapen må på en selvdreven måte kunne gjennomsyre 
+både prosess og produkt. Den typen oppførsel er vanskelig å regulere med mandat og prosjekt. Den kommer gjennom kultur og praktiske opplevelser.
+
+## Fremtind og komponentballen
+Denne læringen tok vi med oss når Fremtind ble formet gjennom en fusjon av SpareBank 1 Forsikring og DNB Forsikring i 2019. 
+Det var derfor innlysende, både for utviklerne, designerne og ledelsen, at vi trengte et designsystem. Men vi trengte også en gruppe med evangelister 
```

Comment:
(Etablere at Fremtind har fått sin egen profil før vi begynner å fordøye den i avsnittet under)

"Ved siden av å håndtere disse to merkevarene, fikk vi nå vår egne identitet utviklet av Scandinavian Design Group. Det var derfor innlysende, både...."

Reply:
Gjorde et forsøk...